src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -47,7 +47,13 @@
     public function load(/*...$paths*/)
```

Comment:
and in the current code, this does not introduce any array_unshift. `$paths = func_get_args();` will still give us *all* arguments